packages/core/src/theme-default/layout/DocLayout/index.tsx

```diff
@@ -83,12 +83,14 @@ export function DocLayout(props: DocLayoutProps) {
           {isOverviewPage ? (
             <Overview />
           ) : (
-            <div className="rspress-doc">
-              <TabDataContext.Provider value={{ tabData, setTabData }}>
-                <MDXProvider components={getCustomMDXComponent()}>
-                  <Content />
-                </MDXProvider>
-              </TabDataContext.Provider>
+            <div>
```

Comment:
Why is there an extra container div?

Reply:
We've removed all child elements of the `﻿.rspress-doc`, and I think the ﻿footer should not be considered part of the article. Thus, I add a `div` to keep `<DocFooter/>`

see: https://github.com/web-infra-dev/rspress/pull/34/files#diff-87b592124a448b2d70388b0cfb70aa567357fbb6140d9de24df2ed6ba5440778R19-R21